### PR TITLE
BAU Fix empty custom branding case

### PR DESCRIPTION
**Pull request by sfount**
Setting a URL of `/` doesn't make our frontend apps happy -- allow empty strings.

### CODE REVIEW

File: src/web/modules/services/branding.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 // eslint-disable-next-line import/prefer-default-export
 export function sanitiseCustomBrandingURL(url: string): string {
-  return url.startsWith('/') ? url : `/${url}`
+  const urlIsAbsolute = url.startsWith('/') || url.length === 0
```
Comment (jankowiakmaria) [inline src/web/modules/services/branding.ts]: Is no url absolute url?
I guess I would prefer rename to something like `shouldPrefixUrl` or something, but can be separate PR.

Comment (sfount) [reply src/web/modules/services/branding.ts]: Arguably an empty string is the base case for an absolute url 👀 can update this if it needs to support more cases or is overly confusing
